introduction_to_amazon_algorithms/imageclassification_mscoco_multi_label/Image-classification-multilabel-lst.ipynb

```diff
@@ -42,9 +42,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    deploy_amt_model = True
</pre><p>I think you added this back in the code block and not as a separate section, as in the other notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>